signUp.html

```diff
@@ -0,0 +1,69 @@
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>FinestStitch</title>
+    <link rel="stylesheet" href="distracted-jones-2f14b5.netlify.app/index.css" />
+    <link rel="stylesheet" href="style.css" />
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.2/css/all.css"
+        integrity="sha384-oS3vJWv+0UjzBfQzYUhtDYW+Pj2yciDJxpsK1OYPAYjqT085Qq/1cq5FLXAZQ7Ay" crossorigin="anonymous">
+</head>
+
+<body>
+    <nav class="navbar">
+        <div class=" logo">
+            <div class="nav-items-container">
+                <a class="nav-item " href="index.html"><span class="logo-font"></span>
+                    <span class="UI-logo-font">FinestStitch</span></a>
+            </div>
+        </div>
+        <div class="topbarCenter">
+            <div class="searchbar">
+                <Search class="searchIcon" />
+                <input placeholder="Search " class="searchInput" />
+            </div>
+        </div>
+        <div class="wishlist-icon">
+            <a href="#"> <i class="fa fa-heart fa-2x nav-icon" aria-hidden="true"></i></a>
+        </div>
+        <div class="cart-icon ">
+            <a href="#">
+                <i class="fa  fa-shopping-cart fa-2x nav-icon" aria-hidden="true"></i></a>
+            <span class=" number-badge number-badge-large">10+</span>
+        </div>
+        <div class="profile-icon dropdown ">
+            <a class="dropbtn" href="#"><i class="fa fa-user  fa-2x nav-icon" aria-hidden="true"></i></a>
+            <div class="dropdown-content">
+                <a class="a" href="login.html"> LogIn </a>
+                <a class="a" href="login.html"> LogOut </a>
+                <a class="a" href="#"> Profile </a>
+            </div>
+        </div>
+    </nav>
+
+    <!-- signUp form sample  -->
+    <div class="login">
+        <div class="signup-Wrapper">
+            <div class="loginRight">
+                <form class="loginBox">
+                    <input placeholder="First Name" type="text" required class="signInInput loginInput" />
+                    <input placeholder="Last Name" type="text" required class="signInInput loginInput" />
+                    <input placeholder="Email" type="email" required class="signInInput loginInput" />
+                    <input placeholder="Password" type="password" required class="signInInput loginInput" />
+                    <input placeholder="Password" type="password" required class="signInInput loginInput" />
+                    <button class="loginButton" type="submit">Sign Up</button>
+                    <span class="loginForgot">Forgot Password?</span>
+                    <button class="loginRegisterButton"><a href="login.html">Already have account?</a></button>
+                </form>
```

Comment:
Suggestion: Wrap the `button` with the `<a>` tag.
As of now the link gets triggered only when clicked on the text which is a bad UX.

Reply:
Done